include/chemfiles/types.hpp

```diff
@@ -173,6 +178,57 @@ inline bool operator!=(const Matrix3D& lhs, const Matrix3D& rhs) {
     return !(lhs == rhs);
 }
 
+/// Negate matrix
+inline Matrix3D operator-(const Matrix3D& lhs) {
+    Matrix3D res;
```

Comment:
Because the default constructor initialize the memory to zero, this might be silghtly slower than calling the constructor directly as 

```cpp
return {
   {-lhs[0][0], -lhs[0][1], -lhs[0][2]},
   {-lhs[1][0], -lhs[1][1], -lhs[1][2]},
   {-lhs[2][0], -lhs[2][1], -lhs[2][2]},
};
```

But the optimizer might be able to figure this.

Reply:
I was not aware of the ability to do that, I just followed the other operators which always created a new Matrix to return.  In any case, I'll merge and close #162 .

Reply:
Yeah, that's fine for now =). I'll try some benchmarks if I get the time

Reply:
That's the only sure fire way to know!